corehq/privileges.py

```diff
@@ -119,6 +119,8 @@
 
 CASE_COPY = 'case_copy'
 
+DEDUPE = 'deduplicate'
```

Comment:
This only applies to cases, right? Would it make sense to make this name a little more specific since there may be other things we deduplicate in the future? For example,
```suggestion
CASE_DEDUPE = 'case_deduplicate'
```

Reply:
Addressed in 9e25684cec33c847162531ee1099775b51e4fb76